backend/geonature/tests/test_gn_meta.py

```diff
@@ -499,6 +500,10 @@ def test_dataset_pdf_export(self, users, datasets):
         assert response.status_code == 200
 
     def test_uuid_report(self, users, synthese_data):
+        observations_nbr = db.session.query(func.count(Synthese.id_synthese)).scalar()
+        if observations_nbr > 1000000:
+            pytest.skip("Too much observations in gn_synthese.synthese")
```

Comment:
xfail, pas skip

Reply:
La nuance est subtile entre `skip` et `xfail`. Il me semble que vis à vis des définitions du début de [cette page](https://docs.pytest.org/en/latest/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail) que c'est plutôt "`skip`".

On veut exécuter le test que si certaines conditions sont présentes (moins d'1 millions d'observations dans la table synthese)...